docs/framework/deployment/windows/installing-dotnet-35-windows-10.md

```diff
@@ -0,0 +1,20 @@
+# Installation the .NET Framework 3.5 on Windows 10
```

Comment:
I think our guidelines say to avoid ing in titles. So it should be Install.

Also, shouldn't we call out the other operating systems?